src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -1,108 +1,108 @@
-            "version": "1.2.0-beta2",
-            "type": "build"
-        },
-        "Roslyn.Diagnostics.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        },
-        "System.Collections.Immutable.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        },
-        //"System.Resources.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
-        "System.Runtime.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        },
-        "System.Runtime.InteropServices.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        },
-        //"System.Security.Cryptography.Hashing.Algorithms.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
-        "System.Threading.Tasks.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        }
-        //"Text.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
-        //"XmlDocumentationComments.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
+    "Roslyn.Diagnostics.Analyzers": {
+      "version": "1.2.0-beta2",
+      "type": "build"
     },
+    "System.Collections.Immutable.Analyzers": {
+      "version": "1.2.0-beta2",
+      "type": "build"
+    },
+    //"System.Resources.Analyzers": {
+    //    "version": "1.2.0-beta2",
+    //    "type": "build"
+    //},
+    "System.Runtime.Analyzers": {
+      "version": "1.2.0-beta2",
+      "type": "build"
+    },
+    "System.Runtime.InteropServices.Analyzers": {
+      "version": "1.2.0-beta2",
+      "type": "build"
+    },
+    //"System.Security.Cryptography.Hashing.Algorithms.Analyzers": {
+    //    "version": "1.2.0-beta2",
+    //    "type": "build"
+    //},
+    "System.Threading.Tasks.Analyzers": {
+      "version": "1.2.0-beta2",
+      "type": "build"
+    }
+    //"Text.Analyzers": {
+    //    "version": "1.2.0-beta2",
+    //    "type": "build"
+    //},
+    //"XmlDocumentationComments.Analyzers": {
+    //    "version": "1.2.0-beta2",
+    //    "type": "build"
+    //},
+  },
 
-    "frameworks": {
-        "net451": {
-            "dependencies": {
-                "Microsoft.ApplicationInsights.DependencyCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.3.0-beta3"
-            }
-        },
-        "netstandard1.6": {
-            "dependencies": {
-                "NETStandard.Library": "1.6.0",
-                "System.Net.NameResolution": "4.0.0"
-            }
-        }
+  "frameworks": {
+    "net46": {
+      "dependencies": {
+        "Microsoft.ApplicationInsights.DependencyCollector": "2.4.0-beta1-build09989",
```

Comment:
The DependencyCollector will not work if we target "net46", because it pick up the dll from the Net45 folder, which is not compatible with the System.Net.Http that is brought by AspNetCore 2.0